lib/snippets/assert-lerna-independent-version-tags

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
```

Comment:
Shellcheck gives me the following report:

```
In lib/snippets/assert-lerna-independent-version-tags line 6:
  HEAD_REV=`git rev-parse HEAD`
           ^-- SC2006: Use $(..) instead of legacy `..`.


In lib/snippets/assert-lerna-independent-version-tags line 13:
  PACKAGE_NAME=$(echo $TAG | cut -d \@ -f 1)
                      ^-- SC2086: Double quote to prevent globbing and word splitting.


In lib/snippets/assert-lerna-independent-version-tags line 14:
  TAG_PACKAGE_VERSION=$(echo $TAG | cut -d \@ -f 2)
                             ^-- SC2086: Double quote to prevent globbing and word splitting.


In lib/snippets/assert-lerna-independent-version-tags line 15:
  ACTUAL_PACKAGE_VERSION=`node -e "console.log(require('./packages/$PACKAGE_NAME/package.json').version)"`
                         ^-- SC2006: Use $(..) instead of legacy `..`.


In lib/snippets/assert-lerna-independent-version-tags line 17:
  if [ $TAG_PACKAGE_VERSION != $ACTUAL_PACKAGE_VERSION ]; then
       ^-- SC2086: Double quote to prevent globbing and word splitting.
                               ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Some good stuff in there.